SEED-platform/seed
# Sidebar menu not refreshed after switching tabs without saving if 'stay on page' option is selected

#### Any background context you want to provide?
1. Sidebar menu tabs' background color doesn't reflect the current page accordingly if user navigates to a different tab without saving changes;
2. The previously navigated page is then disabled if user clicked on it again after choosing to 'stay on page'.

#### What's this PR do?
1. Fixed the sidebar to reflect the current state (dark gray as current v.s. light gray as others);
2. If user decides to stay on page after making changes and navigating to a different tab, this formally clicked on/navigated to tab is reactivated.

#### How should this be manually tested?
1. Go to Data Quality page and make whatever changes;
2. Without clicking on 'save changes' button, click on a different tab on the sidebar, e.g. Data);
3. A prompted window will appear and ask for confirmation for leaving the current page;
4. Click 'stay on page';
5. 'Organization' tab (Data Quality's main page) should stay dark gray color whereas all other tabs are light gray indicating the current state;
6. Now click 'Data' page again, prompt window should show up again and ask for confirmation for leaving current page if changes are still not saved;
7. 'Data' page should be redirected once changes are saved.

#### What are the relevant tickets?
#2066 

#### Screenshots (if appropriate)
